src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -376,6 +377,7 @@
     <AdditionalCleanDirectories Include="@(BinPlaceConfiguration->'%(RuntimePath)')" />
     <AdditionalCleanDirectories Include="@(BinPlaceConfiguration->'%(PackageFileRefPath)')" />
     <AdditionalCleanDirectories Include="@(BinPlaceConfiguration->'%(PackageFileRuntimePath)')" />
+    <AdditionalCleanDirectories Include="@(BinPlaceConfiguration->'%(TestWorkingPath)')" />
```

Comment:
Consider naming this TestPath for parallelism with RefPath and RuntimePath.

Reply:
I will. I just named it different to differentiate it from the <TestPath> property we use in tests.targets